srcpkgs/xbps-triggers/files/binfmts

```diff
@@ -25,26 +25,38 @@ targets)
 	;;
 run)
 	[ -x /usr/bin/update-binfmts ] || exit 0
-	[ -z "${binfmts}" ] && exit 0
+	[ "${binfmts}" ] || [ "${binfmt_files}" ] || exit 0
 
 	case "$TARGET" in
 	post-install)
-		echo "$binfmts" | tr '\' '&' 2> /dev/null | while read line; do
+		printf 'Installing binfmts...\n'
+		printf '%s' "$binfmts" | tr '\' '&' 2> /dev/null | while read line; do
```

Comment:
```
printf '%s\n' ...
```
perhaps?

Otherwise, everything will be concatenated into single long line, and final newline is missing.

Reply:
rebased and addressed this.